bookwyrm/tests/importers/test_goodreads_import.py

```diff
@@ -13,6 +15,10 @@
 from bookwyrm.settings import DOMAIN
 
 
+def make_date(*args):
+    return datetime.datetime(*args, tzinfo=pytz.UTC)
```

Comment:
I don't remember if we like free functions like this, but it seemed just laborious enough to use one to me. I'm not wedded to it tho.

Reply:
I haven't formulated a general opinion on it (seems fine?) but for tests I see absolutely no problem whatsoever. It's handy!